e2e/test.js

```diff
@@ -0,0 +1,49 @@
+const expect = require('chai').expect;
+
+describe('InstantSearch e-commerce demo', () => {
+  before(() => {
+    browser.maximizeWindow();
+  });
+
+  it('must work', async () => {
```

Comment:
For end to end test, I'd tend to use `it` in a slightly different way that in a unit test, for readability of the report and to clarify the intent of the commands that failed.
Example in this case:

```js
it('navigates to the e-commerce demo', () => {
   await browser.url('http://localhost:5000/examples/e-commerce/');
});
it('selects Apple in the brands refinement list', () => {
   const brand = await browser.$('span=Apple');
   await brand.click();
});
```

I'm just suggesting. This test is small enough that it's doesn't beg for it 😄

Reply:
I do not have any opinion on this, it makes the test a bit longer but one advantage of this is that it annotates the test, for example on Sauce Labs you'll get:

![Screen Shot 2019-08-01 at 18 30 52](https://user-images.githubusercontent.com/13209/62311104-56217d80-b48b-11e9-94dc-3c18b9ddc2af.png)

(this is similar to the use of [`sauce:context`](https://wiki.saucelabs.com/display/DOCS/Annotating+Tests+with+Selenium%27s+JavaScript+Executor ))